src/default_panels/StyleAxesPanel.js

```diff
@@ -480,25 +493,12 @@ class StyleAxesPanel extends Component {
           </PlotlySection>
         </AxesFold>
 
-        <AxesFold
-          name={_('Zoom Interactivity')}
-          axisFilter={axis =>
-            !axis._name.includes('angular') && !axis._name.includes('radial')
-          }
-        >
-          <Radio
-            attr="fixedrange"
-            options={[
-              {label: _('Enable'), value: false},
-              {label: _('Disable'), value: true},
-            ]}
-          />
-        </AxesFold>
-
         <AxesFold
           name={_('Hover Projections')}
           axisFilter={axis =>
-            !axis._name.includes('angular') && !axis._name.includes('radial')
+            !axis._name.includes('angular') &&
```

Comment:
is it possible to have `angular` in a `cartesian` subplot? I don't think so... so we can get rid of the first two checks here no?